Resources/config/core.xml

```diff
@@ -27,6 +27,7 @@
         <service id="sonata.admin.breadcrumbs_builder" class="Sonata\AdminBundle\Admin\BreadcrumbsBuilder">
             <argument>%sonata.admin.configuration.breadcrumbs%</argument>
         </service>
+        <service id="sonata.admin.route_id_handler" class="Sonata\AdminBundle\Route\RouteIdHandler"/>
```

Comment:
This should be renamed to `sonata.admin.default_route_id_handler`. `sonata.admin.route_id_handler` should be an alias you would use for your custom implementation.